config/llm.yml

```
@@ -0,0 +1,22 @@
+active_provider: <%= ENV.fetch("LLM_ACTIVE_PROVIDER") { "openai" } %>
+debug: <%= ENV.fetch("AI_DEBUG_MODE") { "false" } %>
+providers:
+  openai:
+    access_token: <%= ENV.fetch("OPENAI_ACCESS_TOKEN") { nil } %>
+    model:  <%= ENV.fetch("OPENAI_MODEL") { nil } %>
```

YAMLlint (1.38.0)

[warning] too many spaces after colon

(colons)